### DISCUSSION TITLE
Unblock Websites with geogebra

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#216] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
There is a way I found, not sure if it works for everyone.

1. Go to webstore
2. Search up GeoGebra, and install it. (If you can't, well then, your fucked. You can just wait for the release of the bookmarklet fix)
3. Go to the three lines in the top right and click them
4. Then, click sign in with google
5. Go to the bottom of the sign in with google popup and click privacy policy
6. scroll all the way to the bottom and click google
7. Congrats! You now have a window that cannot be blocked or viewed!

(This can also be done with sphero EXTENSION, not app. Only if you have installed it previously.)

**Comment by Theblackmamba824** [comment]
so i can install sphero but not geogebra and i dont see a "sign in with google" in the 3 lines

**Reply by ghost** [reply]
hmm thats weird. Its the bigger three lines, not the smaller ones. sign in is at the bottom of the dropdown.
